src/Explorer/Tree/ResourceTree.tsx

```diff
@@ -216,7 +229,8 @@ export const ResourceTree: React.FC<ResourceTreeProps> = ({ container }: Resourc
     const connectGitContextMenu: TreeNodeMenuItem[] = [
       {
         label: "Connect to GitHub",
-        onClick: () =>
+        onClick: () => {
+          container.allocateContainer();
```

Comment:
this should be removed right?

Reply:
This block itself is not being used. Removed